doc/source/building/build_in_buildservice.rst

```diff
@@ -1,27 +1,166 @@
-Building in the Open Buildservice
-=================================
+Building in the Open Build Service
+==================================
 
 .. hint:: **Abstract**
 
    This document gives a brief overview how to build images with
-   KIWI in version |version| inside of the Open Buildservice.
+   KIWI in version |version| inside of the Open Build Service.
    A tutorial on the Open Buildservice itself can be found here:
    https://en.opensuse.org/openSUSE:Build_Service_Tutorial
 
 
-The next generation KIWI is fully integrated with the build service.
+The next generation KIWI is fully integrated with the Open Build Service.
 In order to start it's best to checkout one of the integration test
 image build projects from the base Testing project
-`Virtualization:Appliances:Images:Testing_ARCH` at:
+`Virtualization:Appliances:Images:Testing_$ARCH` at:
 
 https://build.opensuse.org
 
-In order to use the next generation KIWI to build an appliance in the
-build service it is required to add the Builder project as
-repository to the KIWI XML configuration like in the following example:
+For example the test images for x86 can be found `here
+<https://build.opensuse.org/project/show/Virtualization:Appliances:Images:Testing_x86>`__.
+
+
+Advantages of using the Open Build Service (OBS)
+------------------------------------------------
+
+The Open Build Service offers multiple advantages over running KIWI
+locally:
+
+* OBS will host the latest successful build for you without having to setup
+  a server yourself.
+
+* As KIWI is fully integrated into OBS, OBS will automatically rebuild your
+  images if one of the included packages or one of its dependencies or KIWI
+  itself get updated.
+
+* The builds will no longer have to be executed on your own machine, but
+  will run on OBS, thereby saving you resources. Nevertheless, if a build
+  fails, you get a notification via email (if enabled in your user's
+  preferences).
+
+
+Differences Between Building Locally and on OBS
+-----------------------------------------------
+
+Note, there is a number of differences when building images with KIWI using
+the Open Build Service. Your image that build locally just fine, might not
+build without modifications.
+
+The notable differences to running KIWI locally include:
+
+* OBS will pick the KIWI package from the available repositories that you
+  specified in :file:`config.xml`/:file:`config.kiwi`. This means, that you
+  will be very likely building with a different KIWI version in the Open
+  Build Service.
+  This is especially relevant when building images for older versions
+  like SLE. Therefore, include the custom appliances repository as
```

Comment:
Does everybody knows this SLE acronym? Maybe you should better write the complete name (like SUSE Linux Enterprise).